gradlew

```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 
 #
-# Copyright © 2015-2023 the original authors.
+# Copyright © 2015-2021 the original authors.
```

Comment:
???

Reply:
It sure seems like that's what they put in Gradle 8.3? I wonder if that's a bug on their end.

Reply:
Does running lint alter this?

Reply:
The generated gradle script uses 2021. Previously, when I bumped gradle, I manually modified the year because it looked off during the review - no harm in doing that, but we could start just going with whatever gradle uses, which also is a no harm option.